src/content/docs/guides/configure-biome.mdx

```diff
@@ -0,0 +1,391 @@
+---
+title: Configure Biome
+description: Learn how Biome configuration works.
+---
+
+import {FileTree} from '@astrojs/starlight/components';
+
+This guide will help you to understand how to configure Biome.
+It explains the structure of a Biome configuration file and how Biome resolves its configuration.
+If you are already familiar with the configuration, you may want to take a look at the [configuration reference](/reference/configuration/), which details all the options available.
+
+Biome allows you to customize its behavior using CLI options or a configuration file named `biome.json` or `biome.jsonc`.
+We recommend that you create a configuration file for each project.
+This ensures that each team member has the same configuration in the CLI and in any editor that allows Biome integration.
+Many of the options available in a configuration file are also available in the CLI.
+
+## Configuration file structure
+
+A Biome configuration file is named `biome.json` or `biome.jsonc`.
+It is usually placed in your project's root directory, next to your project's `package.json`.
+
+Because Biome is a toolchain, its configuration is organized around the tools it provides.
+At the moment, Biome provides three tools: the formatter, the linter and the import organizer (also called the import sorter).
+All of these tools are enabled by default.
+You can disable one or several of them using the `<tool>.enabled` field:
+
+```json title="biome.json"
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "formatter": {
+    "enabled": false
+  },
+  "linter": {
+    "enabled": false
+  },
+  "organizeImports": {
+    "enabled": false
+  }
+}
+```
+
+Options that apply to any language are placed in the corresponding tool field.
```

Comment:
```suggestion
Options that apply to more than one language are placed in the corresponding tool field.
```

I suggest this because we have https://biomejs.dev/reference/configuration/#formatterattributeposition, which applies to JSX and HTML